Source/Core/Core/ConfigManager.h

```diff
@@ -95,7 +95,7 @@ struct SConfig : NonCopyable
   bool bDSPThread = false;
   bool bDSPHLE = true;
   bool bSyncGPUOnSkipIdleHack = true;
-  bool bForceNTSCJ = false;
+  bool bSelectNTSCAsJapanese = false;
```

Comment:
Another thing: It would be nice if the name didn't imply that the whole console becomes NTSC-J, rather than just the VI bit.